.vsts-ci.yml

```diff
@@ -2,8 +2,23 @@ resources:
 - repo: self
 
 jobs:
+- job: HasRuntimeChanged
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - script: |
+      echo "Determine if runtime changed."
+      git diff --quiet origin/master -- src/rt || echo "##vso[task.setvariable variable=testRuntime;isOutput=true]On" #set variable to testRuntime to On
+      git diff --quiet origin/master -- src/rt && echo "##vso[task.setvariable variable=testRuntime;isOutput=true]Off" #set variable to testRuntime to Off
+      git diff --quiet origin/master -- src/rt && echo "Runtime unchanged!"
+      git diff --quiet origin/master -- src/rt || echo "Runtime changed!"
+    displayName: Check for runtime changes.
+    name: setVarStep
 - job: 
   displayName: Linux
+  dependsOn: HasRuntimeChanged
+  variables: 
+    RTTests: $[ dependencies.HasRuntimeChanged.outputs['setVarStep.testRuntime'] ]
```

Comment:
Can't this be just "ON" and "OFF", to be used on the CMake flags and let CMake do the rest?

For example, if CMake would create a dependency from RT_TEST to "check" target, so that "ninja check" would do all that's required automatically.

Reply:
This is how you pass variables from one job to the next in Azure Dev Ops.  This is just setting up a variable in this job that is either "ON" or "OFF", and it gets passed into cmake using 
```
  -DRT_TESTS=$(RTTests)
```
This then sets up all the build targets and ctest tests to either include or not the runtime tests.